### DISCUSSION TITLE
Use dark mode.

### DISCUSSION THREAD

**Discussion by irakli182** [scaffold-eth/scaffold-eth-2#537] [Q&A] [answered]
Hello, guys. I have noticed that app loads in light mode for about half a sec, then it goes into dark mode. I was wandering, how can I make it load into dark mode? How can I get rid of light mode altogether? thanks!

**Comment by technophile-04** [comment, marked as answer]
Hey @irakli182, you could just remove first object containing [`scaffoldETH`](https://github.com/scaffold-eth/scaffold-eth-2/blob/41b167187c0f5d604f8436431b8fdd126dbb17f5/packages/nextjs/tailwind.config.js#L9) from `themes` array in `tailwind.config.ts` : 

https://github.com/scaffold-eth/scaffold-eth-2/blob/41b167187c0f5d604f8436431b8fdd126dbb17f5/packages/nextjs/tailwind.config.js#L10

Also maybe you should also update Raibowkit default theme to only use dark : 

https://github.com/scaffold-eth/scaffold-eth-2/blob/41b167187c0f5d604f8436431b8fdd126dbb17f5/packages/nextjs/pages/_app.tsx#L41

Hopefully, this helps 🙌
